agents/PCS-Large-LP-Deposit-Withdrawal/src/agent.ts

```diff
@@ -0,0 +1,52 @@
+import { BigNumber } from "ethers";
+import { Finding, HandleTransaction, TransactionEvent, LogDescription, getEthersProvider } from "forta-agent";
+import { EVENTS_ABI, POOL_SUPPLY_THRESHOLD, THRESHOLD_PERCENTAGE, FACTORY } from "./constants";
+import { createPair } from "./utils";
+import { createFinding } from "./finding";
+import PoolFetcher from "./pool.fetcher";
+const stkBNBPoolAddress = "0xaa2527ff1893e0d40d4a454623d362b79e8bb7f1";
+
+export const provideHandleTransaction =
+  (
+    createPair: any,
+    fetcher: PoolFetcher,
+    poolSupplyThreshold: BigNumber,
+    thresholdPercentage: BigNumber,
+    factory: string
+  ): HandleTransaction =>
+  async (txEvent: TransactionEvent): Promise<Finding[]> => {
+    const findings: Finding[] = [];
```

Comment:
Is this working?
You made `findings` constant and pushing/modifying the constant array.
```suggestion
    let findings: Finding[] = [];
```

Reply:
Yup works.